README.md

```diff
@@ -37,17 +37,74 @@ We are willing to hear about your suggestions and improvements.
 
 * Support for CORS protection provided by `flask-cors <https://flask-cors.readthedocs.io/en/latest/>`.
 
+## Python Support
 
-## Installation
+Flask-Philo-Core supports python3.6 or higher. There are not plans to provide
+support older python version.
+
+## Quick Start
+
+### Installation
 
 Flask-Philo-Core installation is pretty straightforward:
 
 ```
 pip3 install Flask-Philo-Core
 ```
 
+### Creating a new project
+
+
+Flask-Philo-Core includes the ``flask-philo-admin`` command line tool.
+
+To quickly generate a new Flask-Philo project, navigate to the directory in which you want to create the project and run:
+
+::
+
+    $ flask-philo-admin startproject
+
+
+This will create a folder called <project_name> which will contain the basic structure of a Flask-Philo-Core application, basic unit tests and configuration.
```

Comment:
Is `<project_name>` a supported argument? Might be worth including it in the example command above